README.md

```diff
@@ -1 +1,182 @@
-# countries-data
\ No newline at end of file
+<div align="center">
+   <img src="./src/images/logo.png" alt="logo" width="auto"  height="auto" />
+  <h3><b>Countries Data</b></h3>
+</div>
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Countries data](#-to-do-list-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [🚀 Live Demo](#live-demo)
+    - [📖 Walk Through](#walk-through)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgements](#acknowledgements)
+  - [📝 License ](#-license-)
+
+# 📖 Countries data<a name="about-project"></a>
+**Countries data** This is a React capstone project about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>Javascript</li>
+    <li>CSS</li>
+    <li>React</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Render a list of categories filtered by some category**
+- **Show detailed data in the category.**
+- **When the user clicks on the "Back" button (<), the user navigates to the home page.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+-[Live Demo Link](https://countries-data-4ptm.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Project Walkthrough <a name="walk-through"></a>
+
+-[Walk through](https://www.loom.com/share/9ad69e78c3444b1d9ebfc45ddfdb33ed)
```

Comment:
- [ ] In your video presentation you failed to show the actual code (in VS Code for example) you are proud of or find interesting. Kindly remake your video including that part.